common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -84,16 +97,23 @@ def test_can_create_and_edit_content_groups(self):
 
         self.assertIn("Updated Second Content Group", second_config.name)
 
-    def test_cannot_delete_content_group(self):
+    def test_can_delete_unused_content_group(self):
         """
-        Scenario: Delete is not currently supported for content groups.
-        Given I have a course without content groups
-        When I create a content group
-        Then there is no delete button
+        Scenario: Ensure that the user can delete unused content group.
+        Given I have a course with 1 Content Group
+        And I go to the Group Configuration page
+        When I delete the Content Group with name "New Content Group"
+        Then I see that there is no Content Group
```

Comment:
I'd add tests for this too: 'When I refresh the page then I see that the content group has been deleted'